src/foundation.cs

```diff
@@ -16086,7 +16086,156 @@ interface NSMeasurementFormatter : NSSecureCoding {
 		string ToString (NSUnit unit);
 	}
 
-	[Watch (2,0), TV (9,0)]
+	[Mac (10, 8)][iOS (6, 0)][Watch (2, 0)][TV (9, 0)]
+	[BaseType (typeof (NSObject), Name = "NSXPCConnection")]
+	[DisableDefaultCtor]
+	interface NSXpcConnection
+	{
+		[Export ("initWithServiceName:")]
+		[NoiOS][NoWatch][NoTV]
+		IntPtr Constructor (string xpcServiceName);
+
+		[Export ("serviceName")]
+		string ServiceName { get; }
+
+		[Export ("initWithMachServiceName:options:")]
+		[NoiOS][NoWatch][NoTV]
+		IntPtr Constructor (string machServiceName, NSXpcConnectionOptions options);
+
+		[Export ("initWithListenerEndpoint:")]
+		IntPtr Constructor (NSXpcListenerEndpoint endpoint);
+
+		[Export ("endpoint")]
+		NSXpcListenerEndpoint Endpoint { get; }
+
+		[Export ("exportedInterface", ArgumentSemantic.Retain)]
+		[NullAllowed]
+		NSXpcInterface ExportedInterface { get; set; }
+
+		[Export ("exportedObject", ArgumentSemantic.Retain)]
+		[NullAllowed]
+		NSObject ExportedObject { get; set; }
+
+		[Export ("remoteObjectInterface", ArgumentSemantic.Retain)]
+		[NullAllowed]
+		NSXpcInterface RemoteInterface { get; set; }
+
+		[Export ("interruptionHandler", ArgumentSemantic.Copy)]
+		Action InterruptionHandler { get; set; }
+
+		[Export ("invalidationHandler", ArgumentSemantic.Copy)]
+		Action InvalidationHandler { get; set; }
+
+		[Export ("resume")]
+		void Resume ();
+
+		[Export ("suspend")]
+		void Suspend ();
+
+		[Export ("invalidate")]
+		void Invalidate ();
+
+		// FIXME: How to bind auditSessionIdentifier?
```

Comment:
`auditSessionIdentifier` is defined as a property whose type is `au_asid_t`: https://developer.apple.com/documentation/foundation/nsxpcconnection/1410393-auditsessionidentifier?language=objc
* `au_asid_t` is a `pid_t`: https://developer.apple.com/documentation/kernel/au_asid_t?language=objc
* `pid_t` is a `__darwin_pid_t`: https://developer.apple.com/documentation/kernel/pid_t?language=objc
* `__darwin_pid_t` is an `__int32_t`: https://github.com/apple/darwin-xnu/blob/a449c6a3b8014d9406c2ddbdc81795da24aa7443/bsd/sys/_types.h#L72

which means that the property can be bound as an int:

```csharp
[Export ("auditSessionIdentifier")]
int AuditSessionIdentifier { get; }
```